examples/stm32/f1/stm32-pill/fancyblink/Makefile

```diff
@@ -0,0 +1,25 @@
+##
+## This file is part of the libopencm3 project.
+##
+## Copyright (C) 2009 Uwe Hermann <uwe@hermann-uwe.de>
```

Comment:
No, this can just be your name, or even just leave it out when they're just stub files like this.  It's definitely not Uwe though.